### PR TITLE
Updated Docker README

### PR DESCRIPTION
**Pull request by Jonhas**
Changed readme.txt to README.md to include proper instructions for creating and mounting a volume to a container.

### CODE REVIEW

**Review by georgebuzzell — APPROVED** [review]
@Jonhas This is FANTASTIC! Thank you for the detailed documentation update here! Just great. :)

**Comment by georgebuzzell** [comment]
@Jonhas  I want to merge your recent edits to the readme file (which are fantastic!) but your PR also includes conflicting commits to delete the Dockerfile and readme.txt?

What is the easiest way to resolve this and merge in? I am new to dealing with conflicting files... so ELI5, please.

@F-said @yanbin-niu if you have any insights here, they are welcome as well!

**Comment by F-said** [comment]
Resolved merge conflicts, accepting to dev.
